src/wallet.ts

```diff
@@ -0,0 +1,216 @@
+import "dotenv/config"
```

Comment:
could we rename this file to zeroDevWallet.ts? want to distinguish between this and ethers wallet which we manage in keystore.ts

Reply:
Updated accordingly